app/helpers/members_helper.rb

```diff
@@ -58,4 +58,15 @@ def member_history_sentence(member)
     end.to_sentence
     text.html_safe + "."
   end
+
+  def member_rebellion(member)
```

Comment:
I think this method could have a more descriptive name, like `member_rebellion_record_sentence` or something like that, so you don't have to look at the code to know what to expect :)